src/js/pages/OpenDispute/index.jsx

```diff
@@ -67,30 +75,34 @@ class OpenDispute extends Component {
       <div className="openDispute">
         <h2>Open dispute</h2>
         <p>Describe details of your trade</p>
-        <Form>
-          <Textarea
-            type="text"
-            name="disputeDetails"
-            id="disputeDetails"
-            rows="7"
-            placeholder="What has happened?"
-            className="form-control mb-2"
-            value={this.state.motive}
-            onChange={this.handleChange}
-            validations={[]}
-          />
-          <p className="text-muted">The process of resolving your dispute could take up to 5 days.</p>
-          <p className="text-center">
-            <Button color="primary" disabled={!this.state.motive} onClick={this.displayDialog(true)}>Send</Button>
-          </p>
-        </Form>
+        <ButtonGroup vertical className="w-100">
+          <CheckButton size="l" active={this.state.motive === UNRESPONSIVE} onClick={this.setMotive(UNRESPONSIVE)}>
+          <b>Unresponsive</b>
+          { isBuyer && <span className="text-muted text-small"><br />you have made the payment but seller is unresponsive</span>}
+          { !isBuyer && <span className="text-muted text-small"><br />buyer has marked trade as paid but is unresponsive and inactive</span>}
+          </CheckButton>
+          <CheckButton size="l" active={this.state.motive === PAYMENT} onClick={this.setMotive(PAYMENT)}>
+          <b>Payment issue</b>
+          { isBuyer && <span className="text-muted text-small"><br />you have made the payment, seller is responsive but states that something is wrong with the payment process, refuses to release</span> }
+          { !isBuyer && <span className="text-muted text-small"><br />buyer is active, has made an attempt to pay, but there are issues with the payment</span>}
+          </CheckButton>
+          <CheckButton size="l" active={this.state.motive === OTHER} onClick={this.setMotive(OTHER)}>
+          <b>Other</b>
+          { isBuyer && <span className="text-muted text-small"><br />any other reason</span> }
```

Comment:
Comment for this whole section: It's better to use `<p>`s instead or `<br/>`s, because they are easier to style and just cleaner.
Anyway, I see you already wrapped your text in `<span>`s, so just need to replace by `<p>`
If you don't want the huge margin under the `<p>`s, you can just put the `mb-0` classname.
Same thing for the bold tags, it's better to do that with CSS, or in our case, with a `<span className="font-weight-bold">`, what way we can control the size.

Last comment, it's prettier to start the sentences with Capital letters